### PR TITLE
Bac backend

### PR DESCRIPTION
**Pull request by WeCaroly**

### CODE REVIEW

File: draught_picks/beers/views.py [hunk 1]
```diff
@@ -92,6 +92,25 @@ class Meta:
         fields = ('uuid', 'user', 'beer', 'created_at',)
 
 
+class BeerWithRecentSerializer(BeerWithRatingSerializer):
+    recents = SerializerMethodField()
+
+    def get_recents(self, obj):
+        req = self.context.get('request')
+        recent = []
```
Comment (jakeharding) [inline draught_picks/beers/views.py]: Doesn't look like we need this `recent` identifier anymore.

File: py_filter.bat [hunk 2]
```diff
@@ -0,0 +1 @@
+doxypypy -a -c %1
```
Comment (jakeharding) [inline py_filter.bat]: Don't think we need this file. Remove it please or add `*.bat` to gitignore and run `git rm --cached py_filter.bat`.

Comment (jakeharding) [reply py_filter.bat]: Doesn't look like it removed this file. Did you run `git rm --cached py_filter.bat` in the terminal?

Comment (WeCaroly) [reply py_filter.bat]: Weird I did run it must have not committed. It'll set it up again.

**Pull request merged**